README.md

```diff
@@ -190,14 +190,6 @@ mount ShopifyApp::Engine, at: '/app-name'
 
 This will create the Shopify engine routes under the specified Subdirectory, as a result it will redirect new consumers to `/app-name/login` and following a similar format for the other engine routes.
 
-To use named routes with the engine so that it can route between the application and the engine's routes it should be prefixed with `main_app` or `shopify_app`.
-
-```ruby
-main_app.login_path # For a named login route on the rails app.
-
-shopify_app.login_path # For the shopify app store login route.
-```
-
```

Comment:
I removed this from the Readme since I think this is more of an internal concern. I can't think of why this would be useful outside of the engine